Makefile.arm64

```diff
@@ -121,7 +121,7 @@ ifeq ($(CORE), NEOVERSEN2)
 ifeq (1, $(filter 1,$(GCCVERSIONGTEQ7) $(ISCLANG)))
 ifeq ($(GCCVERSIONGTEQ9), 1)
 ifeq (1, $(filter 1,$(GCCMINORVERSIONGTEQ4) $(GCCVERSIONGTEQ10)))
```

Comment:
Add gccversion>=11 too ;-)

Reply:
@brada4 11 was >=10  already last time I checked

Reply:
That's basically the same, just on a single line with a more complicated evaluation instead of split into two nested ifeq.
The idea is that we want e.g. gcc 9.4 or better, but the "better" is any higher major number and not just 10.4, 11.4 etc. CMAKE does semantic version checks but gmake needs them spelled out.